mmcv/image/io.py

```diff
@@ -13,15 +13,54 @@
     from cv2 import CV_LOAD_IMAGE_COLOR as IMREAD_COLOR
     from cv2 import CV_LOAD_IMAGE_GRAYSCALE as IMREAD_GRAYSCALE
     from cv2 import CV_LOAD_IMAGE_UNCHANGED as IMREAD_UNCHANGED
+try:
+    from turbojpeg import TJCS_RGB, TJPF_BGR, TJPF_GRAY, TurboJPEG
+except ImportError:
+    TJCS_RGB = TJPF_GRAY = TJPF_BGR = TurboJPEG = None
+
+jpeg = None
+supported_backends = ['cv2', 'turbojpeg']
```

Comment:
This may be moved into the method `use_backend()`.

Reply:
We can export this list, so that user can query them